com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/Constants.java

```diff
@@ -102,7 +102,6 @@
     public static final String VAR_ALL_MODULES_TO_DEPLOY = "allModulesToDeploy";
     public static final String VAR_APPS_TO_DEPLOY = "appsToDeploy";
     // TODO: Do we use this variable?

```

Comment:
Delete this comment as well

Reply:
There are plenty of these comments and I think they are related to "Variables" change.